core/src/main/java/feast/core/job/dataflow/DataflowJobManager.java

```diff
@@ -231,6 +230,78 @@ public JobStatus getJobStatus(Job job) {
     return JobStatus.UNKNOWN;
   }
 
+  @Override
+  public List<Job> listJobs() {
+    List<com.google.api.services.dataflow.model.Job> jobs;
+
+    try {
+      jobs =
+          dataflow
+              .projects()
+              .locations()
+              .jobs()
+              .list(projectId, location)
+              .setFilter("ACTIVE")
+              .execute()
+              .getJobs();
+    } catch (IOException e) {
+      throw new RuntimeException(
+          String.format("Unable to retrieve list of jobs from dataflow: %s", e.getMessage()));
+    }
+
+    return jobs.stream()
+        .map(
+            dfJob -> {
+              try {
+                return dataflow
+                    .projects()
+                    .locations()
+                    .jobs()
+                    .get(projectId, location, dfJob.getId())
+                    .setView("JOB_VIEW_ALL")
+                    .execute();
+              } catch (IOException e) {
+                return null;
```

Comment:
What kind of exceptions are being skipped here?  Should we perhaps be logging a message out?